src/logic/safe/store/actions/processTransaction.ts

```diff
@@ -85,6 +84,8 @@ const processTransaction = ({ approveAndExecute, notifiedTransaction, safeAddres
         dispatch(closeSnackbarAction(beforeExecutionKey))
 
         await saveTxToHistory({ ...txArgs, signature })
+        // TODO: while we wait for the tx to be stored in the service and later update the tx info
```

Comment:
would be great to have a ticket for that

Reply:
https://github.com/gnosis/safe-react/issues/1344